src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatString = sourceString.replace(/;\s*;/g, ' ').trim();
+
+  const stringToArray = formatString.split(';');
+
+  const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `result`. It will decrease code readability. A more descriptive name such as `stylesObject` or `cssProperties` would enhance readability.